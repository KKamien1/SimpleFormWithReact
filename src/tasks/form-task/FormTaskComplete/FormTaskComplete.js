import React from 'react';

export const FormTaskComplete = () => {
    return (
        <>
            <form>
                <fieldset >
                    <legend>Personal Info</legend>
                    <label>First Name: <input type='text' required /></label>
                    <label>Last Name: <input type='text' required /></label>
                    <label>Birthday: <input type='date' /></label>
                </fieldset>

                <fieldset >
                    <legend>User Management</legend>
                    <label>User Type: <input type='text' /></label>
                    <label>User Inactivity Date: <input type='date' /></label>
                </fieldset>
                <button>Save</button>
            </form>
        </>
    );
};
