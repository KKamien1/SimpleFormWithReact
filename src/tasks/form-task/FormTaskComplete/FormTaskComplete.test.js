import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';

import { FormTaskComplete } from './FormTaskComplete'

configure({ adapter: new Adapter() });

describe('FormTaskComplete', () => {

    test('should match snapshot', () => {
        const wrapper = shallow(<FormTaskComplete />);
        expect(wrapper.html()).toMatchSnapshot();
    })

});