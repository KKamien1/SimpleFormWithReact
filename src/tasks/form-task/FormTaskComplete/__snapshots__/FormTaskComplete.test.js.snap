// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTaskComplete should match snapshot 1`] = `"<form><fieldset><legend>Personal Info</legend><label>First Name: <input type=\\"text\\" required=\\"\\"/></label><label>Last Name: <input type=\\"text\\" required=\\"\\"/></label><label>Birthday: <input type=\\"date\\"/></label></fieldset><fieldset><legend>User Management</legend><label>User Type: <input type=\\"text\\"/></label><label>User Inactivity Date: <input type=\\"date\\"/></label></fieldset><button>Save</button></form>"`;
